content/tutorials/reactor-event-ifttt.textile

```diff
@@ -0,0 +1,196 @@
+---
+title: "Reactor Events Tutorial: Integrating your Ably channel with IfThisThenThat"
+alt_title: "Ably-IFTTT integration"
+excerpt: "Learn how to use our Reactor Functions to trigger IFTTT events"
+section: tutorials
+category: external-services
+index: 100
+platform: browser
+authors:
+- author_name: Jo Franchetti
+  author_bio: "Developer advocate at ably"
+  author_profile_url: https://github.com/thisisjofrank
+  author_image: ://avatars2.githubusercontent.com/u/3490640?s=460&v=4
+level: medium
+reading_time: 10
+languages:
+  - javascript
+  - nodejs
+ably_product: reactor
+tags:
+  - ably-features
+---
+
+The Ably Reactor provides a way to trigger events and to stream data from Ably's pub/sub channels. This turorial will go over how to use the Ably Reactor with the "IfThisThenThat Webhooks service":https://ifttt.com/maker_webhooks. IfThisThenThat (IFTTT) allows you to create chains of conditional statements called Applets. These Applets can be triggered by changes to your Ably Channel. They in turn can trigger other web services such as email, social media, chat apps, IoT devices etc.
```

Comment:
Please include links to things you reference from Ably, such as Reactor, pub/sub channels etc, Ably Channel etc.

Reply:
I do no think this is valid terminology: "These Applets can be triggered by changes to your Ably Channel"

The channel is not changing, events or messages are published or emitted on a channel.  The channel itself is not changing.